source/hardware/controllers/akai_apc_mini.rst

```diff
@@ -0,0 +1,28 @@
+Akai APC Mini
+===================
+
+The *Akai APC Mini* is a controller for a 64 Samplers deck.
+
+-  `Manufacturer’s product
+   page <https://www.akaipro.com/apc-mini>`__
+
+Mapping description
+-------------------
+
```

Comment:
Here's are some vector images of the two controllers: Could you edit them in Inkscape and add number labels to the same locations as in the Akai user manuals, and then add lists what each numbered control is supposed to do?

Because for me it's still a bit hard to understand how the mapping is working.
[akai-schematic.zip](https://github.com/mixxxdj/manual/files/5384681/akai-schematic.zip)

Reply:
Sorry for being late, I'll do soon

Reply:
Ping

Reply:
Pong!

Changes done, both images are in with numbers, syntax for buttons added... trailing spaces trimmed. Hope that's all :)